python/paddle/fluid/dygraph/dygraph_to_static/ifelse_transformer.py

```diff
@@ -62,7 +62,6 @@ def transform(self):
         self.after_visit(self.root)
 
     def visit_If(self, node):
-        assert isinstance(node, gast.If)
```

Comment:
Why did you delete this assertion? Is gast.IfExp also goes through visit_If?

Reply:
gast.IfExp will  go through visit_IfExp. I delete it because once enter the `visit_If` function, the node is gast.If exactly. If node is not `gast.If`, this function will not be called.

Reply:
Sounds good to me